README.md

```diff
@@ -145,12 +138,80 @@ for a link-local address is auto-detected.
 
 ## Detailed Description of Capabilities
 
-Here are some more detailes on the additional capabilities of mavlink-router.
-
-### Endpoint Configuration
-
-A description of the different endpoint operating modes and their configuration
-parameters can be found in the [examples/config.sample](examples/config.sample).
+To understand how MAVLink Router forwards messages between endpoints, it
+important to know, that a MAVLink message has to have a sender address, but
+only some messages have a target address (e.g. parameter requests). The sender
+and target address consist of a system and component ID each, so one UAV can
+have a flight controller and companion computer with individual component IDs,
+but sharing the same system ID. In the target address, a component ID or 0 is
+used to broadcast to all components on a system, a sysID of 0 broadcasts to all
+systems.
+
+### Endpoints
+MAVLink Router supports three basic types of endpoints: UART, UDP link and TCP
+client. Additionally, it'll act as a TCP server for dynamic clients (if not
+explicitly deactivated).
+
+Endpoint types (see [examples/config.sample](examples/config.sample) for the
+config file format):
+
+  - UART: For telemetry radios or other serial links
+    * Configuration: UART device path/name and baudrate
+    * Behavior: Data is received and sent right away
```

Comment:
we still "packetize" what we receive, buffering it until we have a complete mavlink message.

Reply:
Rephrased all of the "sent right away" parts.